examples/04_glm_first_level/plot_design_matrix.py

```diff
@@ -111,4 +111,44 @@
 ax2.set_title("Block design matrix", fontsize=12)
 plot_design_matrix(X3, ax=ax3)
 ax3.set_title("FIR design matrix", fontsize=12)
-fig.show()
+plt.show()
+
+
+# %%
+# Parametric modulation
+# ---------------------
+# We may want to modulate the way we model our events in our fMRI analysis.
+# This type of parametric modulation can be done
+# by adding a "modulation" column to the dataframe containing our events.
```

Comment:
How about something like this?

```suggestion
# By default, the fMRI GLM will expect that all events for a given condition have a BOLD
# response with the same amplitude.
# Sometimes, we may have specific expectations about how strong the BOLD response
# will be on a given event.
# This can be incorporated into the model by using **parametric modulation**,
# wherein each event has a predicted amplitude.
# This can be used both to improve model fit and to test hypotheses regarding
# how the BOLD response scales with important features of events,
# such as trial intensity or response time.
```

Reply:
much better !

Reply:
Looks like I didn't manage the line lengths correctly. Sorry about that!

Reply:
well I have given up on making clean code suggestions when using the github UI: I always fix them post hoc anyway